images/istio/tests/main.tf

```diff
@@ -76,6 +76,7 @@ resource "helm_release" "base" {
     global = {
       istioNamespace = local.namespace
     }
+    defaultRevision = ""
```

Comment:
can you add a comment to say you are disabling the install of CRD validation webhook, due to conflict with other tests

Reply:
Done